docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '3.8'
+
+services:
+  app:
+    build: ./go-server
+    ports:
+      - "8080:8080"
+    environment:
+      - DB_URI=mongodb://db:27017
+      - DB_NAME=test
+      - DB_COLLECTION_NAME=todolist
+    depends_on:
+      - db
+
+  client:
+    build: ./client
+    ports:
+      - "3000:3000"
+
+  db:
+    image: mongo:latest
+    ports:
+      - "27017:27017"
```

Comment:
Believe internal networking would be better

Reply:
It's hard to explain, it was swearing at something, that's why I added it.

Reply:
You definitely need to use internal networking. As it's insecure and inappropriate to expose database